pandas/tests/sparse/frame/test_frame.py

```diff
@@ -424,6 +426,11 @@ def _compare_to_dense(a, b, da, db, op):
             sparse_result = op(a, b)
             dense_result = op(da, db)
 
+            if op in [operator.floordiv, ops.rfloordiv] or isinstance(op, LambdaType):
```

Comment:
why is this not just ``callable(op)``?

Reply:
Because we don't want e.g. `operator.add`

Reply:
k can you add that as a comment (future PR ok), as its not obvious

Reply:
sure